### DISCUSSION TITLE
Link are not clickable on the PDF

### DISCUSSION THREAD

**Discussion by amineHY** [marp-team/marp#367] [Q&A] [answered]
Hi
Thank you for this worderfull tool, simple and easy to use

I have a question regarding the links on the generated PDF, they are not clickable, 
is that normal or am I missing an option ?

Thank you in advance

**Comment by yhatt** [comment]
It sounds like strange because hyperlinks on the generated PDF are clickable generally.

```markdown
- URL: https://example.com/
- [Hyperlink](https://example.com/)
```

[👉 PDF](https://github.com/marp-team/marp/files/9994327/untitled.pdf) (every links can click)

Could you share the reproducible example?

**Comment by amineHY** [comment, marked as answer]
Hi @yhatt 
thank you for the reply
I created an example with one link to debug the problem, and as you said, the link clicks normally.
On my presentation, I have generated a Table Of Content automatically, to jump to sections using links quickly, and that was the problem. I had to remove the TOC so the links works normally.
So basically, the link to reference a section in an other page caused the problem some how.
